app/views/partials/_footer.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="card-footer" id="footer">
+  <blockquote class="blockquote text-center mb-0">
+    <p>Kitchen to Couch™</p>
+    <footer class=“blockquote-footer”>Thomas Ochman <cite title=“Source Title”>I don't like carts</cite></footer>
+  </blockquote>
+</div>
```

Comment:
Not sure if wrapping the tag `<footer>` in a div and blockquote is good practice. But I love the quote though.

Reply:
It is a standard bootstrap footer template, we put it in now and will style it when we look at front end, is that wrong to do that?

Reply:
Okay, not a blocker for me.